cmake/modules/RootBuildOptions.cmake

```diff
@@ -481,14 +479,6 @@ endif()
 
 
 #---distributed RDataFrame pyspark tests require both dataframe and pyroot----------------------------------
-if(test_distrdf_pyspark OR test_distrdf_dask)
-
-  if(NOT dataframe OR NOT pyroot)
-    message(STATUS "Running the tests for distributed RDataFrame requires both RDataFrame and PyROOT to be enabled")
-    message(STATUS "    Switching OFF the tests because either pyroot or dataframe option is disabled")
-    message(STATUS "    pyroot is set to ${pyroot} and dataframe is set to ${dataframe}")
-    set(test_distrdf_pyspark OFF CACHE BOOL "Disabled because either dataframe or pyroot were disabled" FORCE)
-    set(test_distrdf_dask OFF CACHE BOOL "Disabled because either dataframe or pyroot were disabled" FORCE)
-  endif()
-
+if(test_distrdf_pyspark OR test_distrdf_dask AND (NOT dataframe OR NOT pyroot))
+    message(FATAL_ERROR "Running the tests for distributed RDataFrame requires both RDataFrame and PyROOT to be enabled")
```

Comment:
I think the previous message was more informative. At least I would add something like 
```
... please build with `-Ddataframe=ON` and `-Dpyroot=ON` to enable these tests
```